doc/api/stream.md

```diff
@@ -1652,6 +1656,10 @@ changes:
 * `destination` {Stream|Function}
   * `source` {AsyncIterable}
   * Returns: {AsyncIterable|Promise}
+* `options` {Object}
+  * `destroyDestOnError` {boolean} If value is false, the destination
+  is not destroyed when a stream error occurs.
+  **Default:** `true`.
```

Comment:
```suggestion
  * `destroyDestOnError` {boolean} If value is `false`, the destination
    is not destroyed when a stream error occurs. **Default:** `true`.
```

Not particularly in love with name `destroyDestOnError` but also don't have any better suggestions.

Reply:
fixed